### ISSUE TITLE
can't name a column created_at in SQLite 

### ISSUE THREAD

**Issue by masukomi** [amberframework/granite-orm#97]
this is related to #29 but can be handled independently

As long as we don't have support for timestamps in SQLite it shouldn't force created_at to be a `Time` format. 

On a related note, if i don't tell granite to use timestamps, it seems silly that it should try and enforce the format of timestamp columns. Yes, it's a bit silly to have "bob" in my `created_at` column, but as I didn't tell granite to do timestamps it shouldn't be enforcing it's opinions on what i can have in that column.

Yes, rails goes ahead and does it, but there's a difference. In rails, I don't have to specify that i want it to use timestamps, which the docs seem to suggest i DO need to do, although this bug seems to be proving the docs wrong.... either that or something is whacky.

In the meantime, i'll be attempting to monkeypatch the created_at functionality out of existence. :/

**Comment by masukomi** [comment]
What follows is my solution, for anyone else encountering this, and also a field guide for devs wanting to clean up the handling of created_at and updated_at which are too smart for their own good and incorrectly setting `SETTINGS[:timestamps] = true` when it wasn't called.

monkeypatch `fields.cr` with this and then call it in your model, because if it sees a `created_at` field it's going to believe that timestamps must be true.
```
module Granite::ORM::Fields
  macro no_timestamps
    {% SETTINGS[:timestamps] = false %}
  end
end
```

then monkeypatch `transactions.cr` 
and alter the `__run_before_save` and `__run_before_create` to stop trying to set updated_at and created at... also get rid of the definitions of those instance variables that sets them to `Time` objects.

then require your new `fields.cr` and `transactions.cr` after you require granite sqlite adapter.

**Comment by drujensen** [comment]
@masukomi thanks for reporting the issue.  

The plan is to support the `created_at` and `updated_at` timestamps once someone takes the time to figure it out.  Looking for help on #29.  

Regarding the issue of it enabling timestamps even though you didn't specify them, I think that is a bug introduced recently.  There should have been a check for the timestamps flag and it should exclude them as the documentation says.

Instead of providing a monkey patch, would you be interested in providing a fix for everyone?  We are always looking for help and your contributions are welcome.

**Comment by drujensen** [comment]
@masukomi are you interested in working on this?

**Comment by damianham** [comment]
fix proposed in https://github.com/amberframework/granite-orm/pull/135

**Comment by drujensen** [comment]
Issue resolved in #135
